client/dashboard/app/snackbars/index.tsx

```diff
@@ -20,6 +20,9 @@ declare module '@tanstack/react-query' {
 			snackbar?: {
 				success?: string;
 				error?: string;
+				speak?: boolean; // announce to screen readers
```

Comment:
This is related to my previous comment. We should definitely P2 about this and ask about designer's input, so let's also separate this to another PR.

A good outcome is maybe to just add these as default values to the snackbar logic, so that devs don't have to think what politeness they should set in their snackbar :)